src/site/stages/build/drupal/graphql/nodeSupportResourcesDetailPage.graphql.js

```diff
@@ -0,0 +1,52 @@
+const entityElementsFromPages = require('./entityElementsForPages.graphql');
+
+const fragment = `
+fragment nodeSupportResourcesDetailPage on NodeSupportResourcesDetailPage {
+  ${entityElementsFromPages}
+  entityBundle
+  changed
+  title
+
+  fieldContentBlock {
+    entity {
+      entityType
+      entityBundle
+      ... wysiwyg
+      ... collapsiblePanel
+      ... process
+      ... qaSection
+      ... qa
+      ... listOfLinkTeasers
+      ... reactWidget
+      ... spanishSummary
+      ... alertParagraph
+      ... table
+      ... downloadableFile
+      ... embeddedImage
+      ... numberCallout
+    }
+  }
+  fieldTableOfContentsBoolean
+  fieldIntroTextLimitedHtml {
+    processed
+  }
+  fieldAlertSingle {
+    entity {
+      ... alertSingle
+    }
+  }
+  fieldButtonsRepeat
+  fieldButtons {
+    entity {
+      ... button
+    }
+  }
+  fieldRelatedInformation {
+    entity {
+      ... listOfLinkTeasers
+    }
+  }
+}
+`;
```

Comment:
I haven't explored it fully yet but I think we need to incorporate `fieldPrimaryCategory`, `fieldOtherCategories` into this. Maybe this is only used internally to the CMS?

```graphql
{
  nodeQuery(limit: 10000, offset: 0, filter: {conditions: [{field: "type", value: ["support_resources_detail_page"]}]}) {
    entities {
      entityId
      entityBundle
      ... on NodeSupportResourcesDetailPage {
        fieldPrimaryCategory {
          entity {
            ... on TaxonomyTermLcCategories {
              name
            }
          }
        }
        fieldOtherCategories {
          entity {
            entityBundle
            ... on TaxonomyTermLcCategories {
              name
            }
          }
        }
      }
    }
  }
}
```

Produces -> 

```json
{
  "data": {
    "nodeQuery": {
      "entities": [
        {
          "entityId": "7910",
          "entityBundle": "support_resources_detail_page",
          "fieldPrimaryCategory": {
            "entity": {
              "name": "VA account and profile"
            }
          },
          "fieldOtherCategories": [
            {
              "entity": {
                "entityBundle": "lc_categories",
                "name": "Other topics and questions"
              }
            }
          ]
        }
      ]
    }
  }
}
```